src/styles/stars.scss

```
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &--4 :nth-child(-n + 4) {
```

Implement the modifier logic here
```suggestion
  .......
  &--3 :nth-child(-n + 3)
  &--4 :nth-child(-n + 4),
  ..... {
```